cmd/ci-operator/main.go

```diff
@@ -270,6 +272,7 @@ func bindOptions(flag *flag.FlagSet) *options {
 	flag.Var(&opt.templatePaths, "template", "A set of paths to optional templates to add as stages to this job. Each template is expected to contain at least one restart=Never pod. Parameters are filled from environment or from the automatic parameters generated by the operator.")
 	flag.Var(&opt.secretDirectories, "secret-dir", "One or more directories that should converted into secrets in the test namespace. If the directory contains a single file with name .dockercfg or config.json it becomes a pull secret.")
 	flag.StringVar(&opt.sshKeyPath, "ssh-key-path", "", "A path of the private ssh key that is going to be used to clone a private repository.")
+	flag.StringVar(&opt.oauthTokenPath, "oauth-token", "", "A path of the OAuth token that is going to be used to clone a private repository.")
```

Comment:
I think we should call this `oauth-token-path` (or just `token-path`) for consistency with `--ssh-key-path`.